torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -1312,6 +1312,80 @@ void nnc_aten_mean(
   }
 }
 
+void nnc_aten_max_pool2d(
+    int64_t bufs_num,
+    void** buf_data,
+    int64_t* buf_ranks,
+    int64_t* buf_dims,
+    int64_t* buf_strides,
+    int8_t* buf_dtypes,
+    int64_t,
+    int64_t* extra_args) {
+  const int64_t x_qdtype = extra_args[2];
+  c10::optional<std::vector<std::pair<size_t, QIData>>> qdata;
+  if (x_qdtype != -1) {
+    qdata = {{1u,
+              {((double*)extra_args)[0],
+               extra_args[1],
+               at::toQIntType(static_cast<c10::ScalarType>(x_qdtype))}}};
+  }
+  auto tensors = constructTensors(
+      bufs_num, buf_data, buf_ranks, buf_dims, buf_strides, buf_dtypes, qdata);
+
+  auto r = at::max_pool2d(tensors[1],
+                          at::IntArrayRef({extra_args[3], extra_args[4]}),
+                          at::IntArrayRef({extra_args[5], extra_args[6]}),
+                          at::IntArrayRef({extra_args[7], extra_args[8]}),
+                          at::IntArrayRef({extra_args[9], extra_args[10]}),
```

Comment:
same readability comment as in https://github.com/Guobing-Chen/pytorch/pull/2

Reply:
Revised with inline comment.